tracks/rust/exercises/macros/mentoring.md

```diff
@@ -22,6 +22,6 @@ macro_rules! hashmap {
             _m
         }
     };
-    ( $($key:expr => $value:expr,)+ ) => { hashmap!($($key => $value),+) }
+    ( $($key:expr => $value:expr,)+ ) => { crate::hashmap!($($key => $value),+) }
```

Comment:
```suggestion
    ( $($key:expr => $value:expr,)+ ) => { $crate::hashmap!($($key => $value),+) }
```

`$crate` means the crate containing the macro definition. `crate` means the crate containing the macro invocation.